src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -336,6 +336,10 @@ void ResumeCtrlImpl::OnSuspend() {
 void ResumeCtrlImpl::OnIgnitionOff() {
   LOG4CXX_AUTO_TRACE(logger_);
   if (!application_manager_.IsLowVoltage()) {
+    if (!resumption_storage_) {
+      LOG4CXX_ERROR(logger_, "resumption_storage_ is not initialized");
+      return;
+    }
```

Comment:
```suggestion
    DCHECK_OR_RETURN_VOID(resumption_storage_);
```
Agreed with @LuxoftAKutsan